share/models/people.py

```diff
@@ -16,25 +16,17 @@ def __str__(self):
         return self.email
 
 
-class Identifier(ShareObject):
```

Comment:
Just chatted with the local SQL expert @cwisecarver. The table's indexes should always be able to fit in memory so it should be fine to put everything in one 👍